__tests__/app.test.js

```diff
@@ -66,14 +66,54 @@ describe("GET: /api/articles", () => {
         });
       });
   });
-  test("200: responds with all of the articles sorted by date in descending order", () => {
+  test("200: responds with all of the articles sorted by date in descending order by default", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body: { articles } }) => {
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("200: responds with all of the articles sorted by the column given", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", { descending: true });
+      });
+  });
+  test("200: responds with all of the articles in the appropriate order when called with the order query", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { ascending: true });
```

Comment:
From reading the[ jest-sorted](https://www.npmjs.com/package/jest-sorted) docs, toBeSortedBy it doesn't seem to have an ascending property. We'd instead get ascending by letting descending default to false (or by explicitly saying false if you think that makes it more clear)